content/features.yaml

```diff
@@ -1,20 +1,16 @@
+- name: interactive
+  title: Interactive, collaborative learning spaces
+  body: Encapsulated learning activities in Activities or Inquiry Learning Spaces (ILS) and can be composed of various elements such as text, images, videos, and even more complex and specific technologies such as apps or labs. All these elements are finally presented in a dynamic and attractive layout and can be shared with a set of students or a wider audience.
   image: 'images/feature-image-1.png'
--
-  name: responsive
-  title: Fully Responsive.
-  body: Aenean condimentum, lacus sit amet luctus lobortis, dolores et quas molestias excepturi enim tellus ultrices elit, amet consequat enim elit noneas sit amet luctu. Quis nostrum exercitationem ullam corporis suscipit laboriosam.Our library is continually refreshed with the latest on web technology so you'll never fall behind. Quis nostrum exercitationem ullam corporis suscipit laboriosam.
+- name: offline
+  title: Offline Support
+  body: Graasp Desktop works while online or offline. Some functionalities such as Visit a Space, Sync and Spaces Nearby require an Internet connection.
   image: 'images/feature-image-2.png'
--
-  name: cross-browser
-  title: Cross-Browser Compatible.
-  body: Lorem ipsum dolor sit amet, ea eum labitur scripserit, illum complectitur deterruisset at pro. Odio quaeque reformidans est eu, expetendis intellegebat has ut, viderer invenire ut his. Has molestie percipit an. Falli volumus efficiantur sed id, ad vel noster propriae. Ius ut etiam vivendo, graeci iudicabit constituto at mea. No soleat fabulas prodesset vel, ut quo solum dicunt. Nec et amet vidisse mentitum. Cibo mutat nulla ei eam.
+- name: user
+  title: User Management
+  body: 'Multiple students can use the same application (on the same computer) and each user can have custom settings, create private and personal (app) input data, set ILSes as favorite or recent (available on the home page), set Favorite ILS: An ILS is set as favorite when the “star” icon is clicked and becomes yellow., set Recent ILS: Recent ILSes are automatically displayed depending on the user’s activities. An ILS is recent if it was visited, saved or loaded.'
   image: 'images/feature-image-3.png'
--
-  name: video
-  title: Video Support.
-  body: Aenean condimentum, lacus sit amet luctus lobortis, dolores et quas molestias excepturi enim tellus ultrices elit, amet consequat enim elit noneas sit amet luctu. Quis nostrum exercitationem ullam corporis suscipit laboriosam. No soleat fabulas prodesset vel, ut quo solum dicunt. Nec et amet vidisse mentitum. Cibo mutat nulla ei eam.
-  vimeo: 14592941
+- name: dashboard
+  title: Dashboard for learning analytics
+  body: A visualization dashboard allowing teachers to access actions more conveniently and providing an overview of their students’ activities. A more compact dashboard is also provided to students so that they can visualize their learning progress.
```

Comment:
A visualization dashboard allows teachers to access actions more conveniently and provides an overview of their students’ activities. A more compact dashboard is also provided to students so that they can follow their learning progress.